enterprise/server/util/ocicache/ocicache_test.go

```diff
@@ -92,3 +82,69 @@ func TestCacheSecret(t *testing.T) {
 		})
 	}
 }
+
+func setupTestEnv(t *testing.T) *testenv.TestEnv {
+	te := testenv.GetTestEnv(t)
+	flags.Set(t, "app.client_identity.client", interfaces.ClientIdentityApp)
+	key, err := random.RandomString(16)
+	require.NoError(t, err)
+	flags.Set(t, "app.client_identity.key", string(key))
+	require.NoError(t, err)
+	err = clientidentity.Register(te)
+	require.NoError(t, err)
+	require.NotNil(t, te.GetClientIdentityService())
+
+	_, runServer, localGRPClis := testenv.RegisterLocalGRPCServer(t, te)
+	testcache.Setup(t, te, localGRPClis)
+	go runServer()
+	return te
+}
+
+// TestManifestWrittenOnlyToAC sets the byte stream server and client to nil,
+// writes a manifest to the AC, and fetches it. If that path were to touch the CAS
+// at all, there would be an error trying to write to a nil client or server.
+func TestManifestWrittenOnlyToAC(t *testing.T) {
```

Comment:
I personally think this test is unnecessary. I don't see how a manifest could unintentionally get written to CAS, but if you'd like to keep it in then that's fine.

Reply:
Going to keep it for now